docs/release.md

```diff
@@ -2,13 +2,14 @@
 
 We use semantic versioning for `obsctl` in format `vX.Y.Z`.
 
-Currently, we do not include binaries with our releases. Instead, users are instructed to install with `go install`.
+We have started including binaries after initial release. Users can also use `go install` to install obsctl.
```

Comment:
```suggestion
We also ship binaries <link to readme>. Users can also use `go install` to install obsctl.
```

Passive voice, we should probably just link to instructions on how to get the binary.

Reply:
Yup, changed, I'll update the readme install section once this change is in, and we have binaries.